openpocket/alpha_spheres/alpha_spheres.py

```diff
@@ -126,27 +125,44 @@ def remove_alpha_spheres(self, indices):
         --------
 
         """
-
-        pass
-
-    def remove_small_alpha_spheres(self, minimum_radius):
-
-        """
-
-        """
-
-        indices_to_remove = np.where(self.radii < minimum_radius)
-        self.remove(indices_to_remove)
+        if indices is not None:
+        #    if isinstance(indices, (list,np.ndarray,dtype:ints)): 
+            if isinstance(indices, (list,np.ndarray)):
+                pass
+            else:
+                raise ValueError("The argument points needs to be a List, tuple or numpy.ndarray object with the integer numbers corresponding to the alpha-sphere")
+
+            self.indices = indices  # a way to store the input indices
+            alpha_sphere_index=[]
+            for ia,alpsp in enumerate(self.points_of_alpha_sphere):
+                alpha_sphere_index.append(ia) 
+            for ix in self.indices:
+#                if ix in alpha_sphere_index:
+                alpha_sphere_index.remove(ix)
+               # remaining=[]
+                self.remove_alpha_spheres=[]
```

Comment:
There is no need to have a list of `remove_alpha_spheres`. The goal of this function is to remove entries from the attributes `self.centers`, `self.points_of_alpha_sphere` and `self.radii`, as well as to adjust the value of `self.n_alpha_spheres`.

Any way... if you want to create a new attribute in the class named `remove_alpha_spheres`, why do you do it every iteration of `for ix in self.indices:`?

Reply:
It's the same mistake as before.